File: Baekjoon/solved_ac/class1/2577.py
# 2577: 숫자의 개수
num = 1

cnt_list = [0 for _ in range(10)]

for i in range(3):
  num *= int(input())

for i in str(num):
  cnt_list[int(i)] += 1

for cnt in cnt_list:
  print(cnt)

File: Baekjoon/solved_ac/class1/2577.cpp
// 2577: 숫자의 개수

# include <iostream>
# include <string>

int main() {
  int cnt_list[10] = {0, };
  int num = 1;

  for (int i=0; i<3; i++) {
    int input_num;
    std::cin >> input_num;
    num *= input_num;
  }
  
  std::string str_num = std::to_string(num);
  for (char c : str_num) {
    cnt_list[c - '0']++;
  }

  for (int i=0; i<10; i++) {
    std::cout << cnt_list[i] << std::endl;
  }  

  return 0;
}